app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+    <%= f.submit "Log in", class:"login-button" %>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Kindly design the login form to match the wireframe. The elements should be centered per the wireframe layout

  Your Implementation | Sneek Peek Design
  :---:|:---:
  ![log-ric](https://github.com/RicardoEgas/blog-app/assets/54848363/23812bb6-7d29-4c75-8b5c-d125da461edf) | ![wireframe-ric](https://github.com/RicardoEgas/blog-app/assets/54848363/aac514b9-5ee7-4e12-8211-7ad8035f0801)